src/kontrol/prove.py

```diff
@@ -652,66 +657,81 @@ def _method_to_cfg(
-    number_cell = node.cterm.cell('NUMBER_CELL')
-    timestamp_cell = node.cterm.cell('TIMESTAMP_CELL')
-    basefee_cell = node.cterm.cell('BASEFEE_CELL')
-    chainid_cell = node.cterm.cell('CHAINID_CELL')
-    coinbase_cell = node.cterm.cell('COINBASE_CELL')
-    prevcaller_cell = node.cterm.cell('PREVCALLER_CELL')
-    prevorigin_cell = node.cterm.cell('PREVORIGIN_CELL')
-    newcaller_cell = node.cterm.cell('NEWCALLER_CELL')
-    neworigin_cell = node.cterm.cell('NEWORIGIN_CELL')
-    active_cell = node.cterm.cell('ACTIVE_CELL')
-    depth_cell = node.cterm.cell('DEPTH_CELL')
-    singlecall_cell = node.cterm.cell('SINGLECALL_CELL')
-    gas_cell = node.cterm.cell('GAS_CELL')
-    callgas_cell = node.cterm.cell('CALLGAS_CELL')
-
-    all_accounts = flatten_label('_AccountCellMap_', new_accounts_cell)
-
-    new_accounts = [CTerm(account, []) for account in all_accounts if (type(account) is KApply and account.is_cell)]
-    non_cell_accounts = [account for account in all_accounts if not (type(account) is KApply and account.is_cell)]
-
-    new_accounts_map = {account.cell('ACCTID_CELL'): account for account in new_accounts}
-
-    if config_type == ConfigType.SUMMARY_CONFIG:
-        for account_id, account in new_accounts_map.items():
-            acct_id_cell = account.cell('ACCTID_CELL')
-            if type(acct_id_cell) is KVariable:
-                acct_id = acct_id_cell.name
-            elif type(acct_id_cell) is KToken:
-                acct_id = acct_id_cell.token
-            else:
-                raise RuntimeError(
-                    f'Failed to abstract storage. acctId cell is neither variable nor token: {acct_id_cell}'
-                )
-            new_accounts_map[account_id] = CTerm(
-                set_cell(
-                    account.config,
-                    'STORAGE_CELL',
-                    KVariable(f'STORAGE_{acct_id}', sort=KSort('Map')),
-                ),
-                [],
-            )
+    if config_type == ConfigType.TEST_CONFIG:
+        new_accounts_cell = node.cterm.cell('ACCOUNTS_CELL')
+        number_cell = node.cterm.cell('NUMBER_CELL')
+        timestamp_cell = node.cterm.cell('TIMESTAMP_CELL')
+        basefee_cell = node.cterm.cell('BASEFEE_CELL')
+        chainid_cell = node.cterm.cell('CHAINID_CELL')
+        coinbase_cell = node.cterm.cell('COINBASE_CELL')
+        prevcaller_cell = node.cterm.cell('PREVCALLER_CELL')
+        prevorigin_cell = node.cterm.cell('PREVORIGIN_CELL')
+        newcaller_cell = node.cterm.cell('NEWCALLER_CELL')
+        neworigin_cell = node.cterm.cell('NEWORIGIN_CELL')
+        active_cell = node.cterm.cell('ACTIVE_CELL')
+        depth_cell = node.cterm.cell('DEPTH_CELL')
+        singlecall_cell = node.cterm.cell('SINGLECALL_CELL')
+        gas_cell = node.cterm.cell('GAS_CELL')
+        callgas_cell = node.cterm.cell('CALLGAS_CELL')
+
+        all_accounts = flatten_label('_AccountCellMap_', new_accounts_cell)
```

Comment:
I think that this function can be refactored a bit to avoid repetition and improve clarity:

You could create a list with all the cell names, and a dict with the cells.

```suggestion
    cell_names = [
        'ACCOUNTS_CELL', 'NUMBER_CELL', 'TIMESTAMP_CELL', 'BASEFEE_CELL', 
        'CHAINID_CELL', 'COINBASE_CELL', 'PREVCALLER_CELL', 'PREVORIGIN_CELL',
        'NEWCALLER_CELL', 'NEWORIGIN_CELL', 'ACTIVE_CELL', 'DEPTH_CELL', 
        'SINGLECALL_CELL', 'GAS_CELL', 'CALLGAS_CELL'
    ]
    
    cells = {name: node.cterm.cell(name) for name in cell_names}
    all_accounts = flatten_label('_AccountCellMap_', cells['ACCOUNTS_CELL'])
```

Reply:
Alternatively, you can drop the `cell_*` variables and use `node.cterm.cell(cell_name)` directly in the loop I suggested below. In this case, you would have to process `all_accounts` independently.